### PR TITLE
Increase default maximum size of ipsets and make configurable.

### LINKED ISSUE #768: Felix dies if ipset gets too big
An ipset can hold many entries (maybe ~65k, by default) but we don't put a limit on how many entries we'll try to put in one.  If the the ipset restore call fails due to the number of entries being exceeded then felix dies with 

```
<snip>
add felix-tmp-v4-prof-000 1.1.68.108
add felix-tmp-v4-prof-000 1.1.68.107
add felix-tmp-v4-prof-000 1.1.68.106
add felix-tmp-v4-prof-000 1.1.68.105
add felix-tmp-v4-prof-000 1.1.68.104
add felix-tmp-v4-prof-000 1.1.68.103
add felix-tmp-v4-prof-000 1.1.68.102
add felix-tmp-v4-prof-000 1.0.50.108
add felix-tmp-v4-prof-000 1.1.68.100
swap felix-v4-prof-000 felix-tmp-v4-prof-000
destroy felix-tmp-v4-prof-000
COMMIT


TrackedAsyncResult ('ipsets.py:100:_update_active_ipset', 'IpsetManager(IPv4) (processing 6a605efe775a)', 'TagIpset(prof-000)', 'replace_members') was leaked with exception FailedSystemCall('ipset', 'restore').  Dying.
```

I'm not sure if the limit can be raised (so that we can dynamically choose to increase it once we're getting close) or if it's a hard limit (in which case we should isolate the failure to the single ipset and log an error).

### PR DESCRIPTION
**Pull request by fasaxc**
Fixes #768.

### CODE REVIEW

**Comment by fasaxc** [comment]
retest this please

**Comment by fasaxc** [comment]
FV failure is due to FV test problem, should be resolved when the development branch is merged in later.

**Comment by fasaxc** [comment]
retest this please

File: calico/felix/test/test_ipsets.py [hunk 1]
```diff
@@ -518,16 +539,46 @@ def setUp(self):
...
-            input_str='create foo hash:ip family inet --exist\n'
-                      'create foo-tmp hash:ip family inet --exist\n'
-                      'flush foo-tmp\n'
-                      'add foo-tmp 10.0.0.1\n'
-                      'swap foo foo-tmp\n'
-                      'destroy foo-tmp\n'
-                      'COMMIT\n'
-        )
+        exp_calls = [
+            call(["ipset", "destroy", "foo-tmp"]),
```
Comment (matthewdupre) [inline calico/felix/test/test_ipsets.py]: Note to self: figure out why we've gained these destroy calls.  Why are they in a separate commit?  Do we still need the flush?

**Comment by matthewdupre** [comment]
OK, found the comments explaining the delete.  Looks good, but I would like to see a successful FV.

**Comment by matthewdupre** [comment]
retest this please

**Comment by matthewdupre** [comment]
retest this please

**Comment by matthewdupre** [comment]
retest this please

**Comment by fasaxc** [comment]
retest this please

**Comment by fasaxc** [comment]
Another FV failure due to a timeout at install time.

**Pull request merged**